src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -263,12 +273,30 @@ public boolean hasMapping(String index) {
    * document id when either the DLQ is configured or
    * {@link ElasticsearchSinkConnectorConfig#IGNORE_KEY_CONFIG} is set to <code>false</code> because
    * they require the use of a map keyed by document id.
+   * <br>
+   * This call is usually asynchronous, but can block if:
```

Comment:
Nit: Please clarify whether this call can block if *both* or *either* of the following conditions are true.

Reply:
Added